### PR TITLE
Updated the names in Egg Quiz and Word Window Urdu TSVs

### PR DESCRIPTION
**Pull request by bikashtudu**
|PR checklist|Status| 
|-----|:------:| 
|Is issue created in GitHub?|[Yes](https://github.com/maqsoftware/Pehla-School/issues/324)| 
|Are the changes replicated across different locales?|NA| 
|Is code analysis task executed and issues resolved in the Java/Kotlin code?|NA| 
|Is code analysis task executed and issues resolved in the CPP/native code?|NA| 
|Is the version code set in the AndroidManifest.xml or build.gradle file to 1 more than the version which has already been published?|NA| 
|Is code commenting done properly for the changes made?|NA| 
|Is correct variable naming convention followed?|NA| 
|Is auto formatting shortcut Ctrl+Alt+Shift+L executed in all files?|NA| 
|Are the message strings used in the application taken from strings.xml file?|NA| 
|Are the permissions which are not required removed from AndroidManifest.java file?|NA| 
|Is the _uses-feature_ tag used for only must have feature in AndroidManifest.java file?|NA| 
|Are the generated files correctly configured in .gitignore?|NA| 
|Are the release signing configurations removed from the gradle files?|NA| 
|Is commented code removed from all files?|NA| 
|Are updated images compressed (< 100 KB)?|NA| 
|Ensured there are no typos in any string. For example: Variable/method names, hardcoded strings, commit messages, PR titles, issues, code comments?|NA|

### CODE REVIEW

File: localized/games/eggquiz/eggquizliteracy_levels_ur.tsv [hunk 1]
```diff
@@ -813,7 +813,7 @@ SpecialTest_12	1	120001, 120002, 120003, 120004, 120005
 			120002	soundonly_word					house	house	happy^road^house
 			120003	soundonly_word					girl	girl	man ^girl^boy
 			120004	sentence_word		______ are tall.			you are tall	You	I^He^You
-			120005	sentence_word		We ______ Mike.			we like mike	like	like^walk ^want
+			120005	sentence_word		We ______ Mubin.			we like mike	like	like^walk ^want
```
Comment (Shailin) [inline localized/games/eggquiz/eggquizliteracy_levels_ur.tsv]: `we like mike` is not changed. Make sure all words are replaced

File: localized/games/eggquiz/eggquizliteracy_levels_ur.tsv [hunk 2]
```diff
@@ -880,17 +880,17 @@ SpecialTest_16	2	160006, 160007, 160008, 160009, 160010
 											
 SpecialTest_17	1	170001, 170002, 170003, 170004, 170005									
 			170001	sentence_word		____ are tall.			you are tall	You	you^YoU^You
-			170002	sentence_word		____ like Josh.			I like josh	I	i^I ^iI
+			170002	sentence_word		____ like Jeesh.			I like josh	I	i^I ^iI
```
Comment (Shailin) [inline localized/games/eggquiz/eggquizliteracy_levels_ur.tsv]: Replace `josh` with `Jeesh`.

File: localized/games/eggquiz/eggquizliteracy_levels_ur.tsv [hunk 3]
```diff
@@ -880,17 +880,17 @@ SpecialTest_16	2	160006, 160007, 160008, 160009, 160010
...
 			170003	sentence_sentence		صحیح جواب چنیں			सही चयन करे	He is a boy.	he is a boy.^He is a boy.^He is A BOy.
 			170004	sentence_sentence		صحیح جواب چنیں			सही चयन करे	Birds fly.	Birds fly.^birds fly.^biRds flY.
-			170005	sentence_sentence		صحیح جواب چنیں			सही चयन करे	This is Ann.	this is ann.^This is Ann.^this is Ann.
+			170005	sentence_sentence		صحیح جواب چنیں			सही चयन करे	This is Amreen.	this is ann.^This is Amreen.^this is Amreen.
```
Comment (Shailin) [inline localized/games/eggquiz/eggquizliteracy_levels_ur.tsv]: Replace word `ann` with `Amreen`

File: localized/games/eggquiz/eggquizliteracy_levels_ur.tsv [hunk 4]
```diff
@@ -1239,14 +1239,14 @@ FishTest_6_2	1	6200001, 6200002, 6200003, 6200004, 6200005		sentence_sentence		
 			6200002	sentence_word		صحیح جواب چنیں			heart	heart	hiert^heart^hart
 			6200003	soundonly_word					play	play	fly^clay^play
 			6200004	sentence_word		This is my ____.			this is my brother	brother	brother^mother^dad
-			6200005	sentence_word		Fred is in his _____.			fred is in his bedroom	bedroom	bedroom^kitchen^house
+			6200005	sentence_word		Fareed is in his _____.			fred is in his bedroom	bedroom	bedroom^kitchen^house
```
Comment (Shailin) [inline localized/games/eggquiz/eggquizliteracy_levels_ur.tsv]: Replace `fred` with `fareed`

File: localized/games/eggquiz/eggquizliteracy_levels_ur.tsv [hunk 5]
```diff
@@ -1239,14 +1239,14 @@ FishTest_6_2	1	6200001, 6200002, 6200003, 6200004, 6200005		sentence_sentence		
...
 											
 FishTest_6_2	2	6200006, 6200007, 6200008, 6200009, 6200010									
 			6200006	sentence_word		صحیح جواب چنیں			news	news	mews^vews^news
 			6200007	sentence_word		صحیح جواب چنیں			paper	paper	paper^peper^papir
 			6200008	soundonly_word					sweet	sweet	sweet^green^brown
 			6200009	sentence_word		This is my ____.			this is my mother	mother	brother^mother^dad
-			6200010	sentence_word		Cindy is in the _____.			cindy is in the kitchen	kitchen	bedroom^kitchen^house
+			6200010	sentence_word		Chanda is in the _____.			cindy is in the kitchen	kitchen	bedroom^kitchen^house
```
Comment (Shailin) [inline localized/games/eggquiz/eggquizliteracy_levels_ur.tsv]: Please properly review the changes made. The word is not replaced at many places

File: localized/games/eggquiz/eggquizmath_levels_ur.tsv [hunk 6]
```diff
@@ -161,13 +161,13 @@ MiniTest_7_2	1	72001, 72002, 72003, 72004, 72005
...
+			72005	word_problem	بختیار كے پاس 2 مربلیس ہیں . علیزہ كے پاس بختیار سے 3 زیادہ مربلیس ہیں . علیزہ كے پاس کل کتنے مربلیس ہیں ؟	बख़्तियार के पास two मार्बल्स हैं। ऑड्रे के पास बख़्तियार की तुलना में three अधिक मार्बल्स हैं। ऑड्रे के पास कितने मार्बल्स हैं?				5				
 MiniTest_7_3	1	73001, 73002, 73003, 73004, 73005												
 			73001	compare_number_magnitudes	ascending	اعداد کو چھوٹے سے بڑے کی ترتیب دیں	सबसे छोटे से लेकर सबसे बड़े तक के क्रम में संख्या को रखे			71~99				
 			73002	missing_number_drag	2	3	descending			11~30	2			
 			73003	operations_with_objects	11~19+11~19=?	regroupingtrue				20~40				
 			73004	operations_drag	21~50-10~20=?	regroupingfalse	vertical			1~40				
-			73005	word_problem	5 ٹیبل پر 5 سب رکھے ہیں . سام 2 سب کھا لیتا ہے . ٹیبل پر اب کتنے سب بچے ہیں ؟	Five एपल्स टेबल पर हैं। सैम two एपल्स खाता है। टेबल पर कितने सेब बचे हैं?				3				
+			73005	word_problem	5 ٹیبل پر 5 سب رکھے ہیں . سام 2 سب کھا لیتا ہے . ٹیبل پر اب کتنے سب بچے ہیں ؟	Five एपल्स टेबल पर हैं। सलमान two एपल्स खाता है। टेबल पर कितने सेब बचे हैं?				3
```
Comment (Shailin) [inline localized/games/eggquiz/eggquizmath_levels_ur.tsv]: Urdu text is not updated

File: localized/games/eggquiz/eggquizmath_levels_ur.tsv [hunk 7]
```diff
@@ -178,46 +178,46 @@ PostTest_7	1	7001, 7002~7003, 7004, 7005, 7006~7007, 7008, 7009, 7010~7011, 7012
 			7007	operations_with_objects	1~9+11~19=?	regroupingtrue				20~30				
 			7008	operations_with_objects	11~19+11~19=?	regroupingtrue				20~40				
 			7009	operations_drag	11~50-11~30=?	regroupingfalse	vertical			1~50				
-			7010	word_problem	اسٹاکی نے صبح 5 مچھلیاں پکڑی. اس نے دوپہر میں 4 اور مچھلیاں پکڑی. اسٹاکی نے سب ملکر کتنی مچھلیاں پکڑی؟	स्टैसी ने सुबह 5 मछलियाँ पकड़ीं। उसने दोपहर में 4 और मछलियाँ पकड़ीं। स्टैसी ने कुल कितनी मछली पकड़ी?				9				
-			7011	word_problem	ہینری كے پاس 4 اسٹیمپس ہیں. ایرک كے پاس ہینری سے 5 اسٹیمپس زیادہ ہیں. ایرک كے پاس کتنے اسٹیمپس ہیں؟	हेनरी के पास 4 डाक टिकट हैं। एरिक के पास हेनरी की तुलना में 5 अधिक डाक टिकट हैं। एरिक के पास कितने डाक टिकट हैं?				8				
-			7012	word_problem	ہینری كے پاس 4 اسٹیمپس ہیں . ایرک كے پاس ہینری سے 5 زیادہ ڈال اسٹیمپس ہیں . ایرک كے پاس کتنے ستماپس ہیں ؟	एक पेड़ की शाखा पर 8 पक्षी बैठे हैं। कुछ उड़ गए। अब शाखा पर 3 पक्षी हैं। कितने पक्षी उड़ गए?				3				
+			7010	word_problem	صابرہ نے صبح 5 مچھلیاں پکڑی. اس نے دوپہر میں 4 اور مچھلیاں پکڑی. صابرہ نے سب ملکر کتنی مچھلیاں پکڑی؟	स्टैसी ने सुबह 5 मछलियाँ पकड़ीं। उसने दोपहर में 4 और मछलियाँ पकड़ीं। स्टैसी ने कुल कितनी मछली पकड़ी?				9
```
Comment (Shailin) [inline localized/games/eggquiz/eggquizmath_levels_ur.tsv]: Replace word `स्टैसी`

File: localized/games/eggquiz/eggquizmath_levels_ur.tsv [hunk 8]
```diff
@@ -178,46 +178,46 @@ PostTest_7	1	7001, 7002~7003, 7004, 7005, 7006~7007, 7008, 7009, 7010~7011, 7012
...
+			82005	word_problem	ایک باڑ پر 12 بلیاں ہیں. 1 بختیار باڑ کے نیچے چھلانگ لگاتی ہے. کتنی بلیوں کو باڑ پر چھوڑ دیا جاتا ہے ؟	एक बाड़ पर 12 बिल्लियाँ हैं। 1 बिल्ली बाड़ से नीचे कूदती है। अब कितनी बिल्लियाँ बाड़ पर हैं?				11				
 MiniTest_8_3	1	83001, 83002, 83003, 83004, 83005												
 			83001	missing_number_drag	100	2	ascending			100~900	100			
 			83002	operations_drag	11~18-1~9=?	regroupingtrue	horizontal			1~9				
 			83003	operations_drag	21~50-11~30=?	regroupingtrue	vertical			1~40				
-			83004	word_problem	جینٹ ہے 8 نارنگیاں اور شیرون ہے 4 نارنگیاں. کتنے نارنگیاں کرتے ہیں وہ مکمل طور پر ؟	जेनेट के पास 8 संतरे हैं और शेरोन के पास 4 संतरे हैं। उनके पास कुल मिलाकर कितने संतरे हैं?				12				
-			83005	word_problem	سینڈی کی عمر 5 سال ہے۔ میا کی عمر سینڈی سے 7 سال زیادہ ہے۔ میا کی عمر کتنی ہے؟	सैंडी की उम्र 5 साल है। मिया सैंडी से 7 साल बड़ी है। मिया कितनी साल की है?				12				
+			83004	word_problem	جمیلہ ہے 8 نارنگیاں اور ساجد ہے 4 نارنگیاں. کتنے نارنگیاں کرتے ہیں وہ مکمل طور پر ؟	जेनेट के पास 8 संतरे हैं और शेरोन के पास 4 संतरे हैं। उनके पास कुल मिलाकर कितने संतरे हैं?				12
```
Comment (Shailin) [inline localized/games/eggquiz/eggquizmath_levels_ur.tsv]: Replace word `जेनेट`

File: localized/games/eggquiz/eggquizmath_levels_ur.tsv [hunk 9]
```diff
@@ -178,46 +178,46 @@ PostTest_7	1	7001, 7002~7003, 7004, 7005, 7006~7007, 7008, 7009, 7010~7011, 7012
...
 			8006	operations_drag	31~50-11~20=?	regroupingtrue	vertical			1~50				
-			8007	word_problem	ٹام کے پاس 6 کتابیں ہیں. جیمز کے پاس دو کتابیں ہیں.سب ملکر وہ کتنی کتابیں ہیں ؟	टॉम के पास 6 किताबें हैं। जेम्स के पास 2 किताबें हैं। उनके पास सभी में कितनी किताबें हैं?				8				
-			8008	word_problem	ڈیاگو کے پاس ایک ٹوکری میں 11 مانگوس ہے. ڈیاگو اپنی بہن کو 4 مانگوس دیتا ہے. کتنی مانگوس ٹوکری میں رہ رہے ہیں ؟	डिएगो के  एक टोकरी में 11 आम हैं। डिएगो अपनी बहन को 4 आम देता है। टोकरी में कितने आम बचे हैं?				7				
-			8009	word_problem	فریڈ کی 12 سال کی عمر ہے. ان کے بھائی ٹام کی 8 سال کی عمر ہے. کتنے چھوٹے فریڈ سے ٹام ہے?	फ्रेड 12 साल का है। उसका भाई टॉम 8 साल का है। फ्रेड की तुलना में टॉम कितना छोटा है?				4				
+			8007	word_problem	طالب کے پاس 6 کتابیں ہیں. جابر کے پاس دو کتابیں ہیں.سب ملکر وہ کتنی کتابیں ہیں ؟	तालिब के पास 6 किताबें हैं। जेम्स के पास 2 किताबें हैं। उनके पास सभी में कितनी किताबें हैं?				8				
+			8008	word_problem	ارمان کے پاس ایک ٹوکری میں 11 مانگوس ہے. ارمان اپنی بہن کو 4 مانگوس دیتا ہے. کتنی مانگوس ٹوکری میں رہ رہے ہیں ؟	डिएगो के  एक टोकरी में 11 आम हैं। डिएगो अपनी बहन को 4 आम देता है। टोकरी में कितने आम बचे हैं?				7
```
Comment (Shailin) [inline localized/games/eggquiz/eggquizmath_levels_ur.tsv]: Replace word `डिएगो`

File: localized/games/eggquiz/eggquizmath_levels_ur.tsv [hunk 10]
```diff
@@ -388,37 +388,37 @@ SpecialTest_21	1	210001, 210002, 210003, 210003, 210004
...
-			220008	word_problem	انا كے پاس 7 بسکٹ ہیں . وہ اس میں سے 2 کھا لیتی ہے. اب اسکے پاس کتنے بسکٹ بچے ہیں؟	एना के पास 7 बिस्कुट हैं। वह उनमें से 2 खाती है। उसके पास कितनी बिस्कुट बचे हैं?				5
+			220005	word_problem	لبنا کے پاس 12 کتابیں ہیں ۔ وہ اس کی کلاس کی لائبریری میں عطیہ کرتی ہے 8 کتابیں. کتنی کتابیں لبنا کے پاس ہیں ؟	लीना के पास 12 किताबें हैं। वह अपनी कक्षा की लाइब्रेरी में 8 किताबें दान करती हैं। लीना के पास अब कितनी किताबें बची हैं?				4
+			220006	word_problem	ایک باڑ پر 12 بلیاں ہیں. 1 بختیار باڑ کے نیچے چھلانگ لگاتی ہے. کتنی بلیوں کو باڑ پر چھوڑ دیا جاتا ہے ؟	एक बाड़ पर 12 बिल्लियाँ हैं। 1 बिल्ली बाड़ से नीचे कूदती है। अब कितनी बिल्लियाँ बाड़ पर हैं?				11
+			220007	word_problem	لیری كے پاس 8 پنسل ہے. وہ ان میں سے 7 کھو دیتا ہے .لیری كے پاس کتنی پنسل بچی ہے؟	आसमा के पास 8 पेंसिल हैं। वह उनमें से 7 को खो देती है। आसमा के पास अब कितनी पेंसिल बची हैं?				1
+			220008	word_problem	انم كے پاس 7 بسکٹ ہیں . وہ اس میں سے 2 کھا لیتی ہے. اب اسکے پاس کتنے بسکٹ بچے ہیں؟	एना के पास 7 बिस्कुट हैं। वह उनमें से 2 खाती है। उसके पास कितनी बिस्कुट बचे हैं?				5
```
Comment (Shailin) [inline localized/games/eggquiz/eggquizmath_levels_ur.tsv]: Properly check Urdu and Hindi text. Either Hindi or Urdu text is not updated at many places

File: localized/games/wordwindow/wordwindow_level_ur.tsv [hunk 11]
```diff
@@ -1,241 +1,241 @@
...
+en-US	9	2	1	6	"टेस ने सुबह 3 फूल उठाए। उसने दोपहर मे कुछ और फूल उठाए। यदि टेस ने सभी मे 8 फूल उठाए, तो उसने दोपहर मे कितने उठाए?"	طوبا نے صبح میں 3 پھول اٹھاے. اسنے دوپہر میں کچھ پھول اور اٹھاے. اگر اسنے کل ملا کر  8 پھول اٹھاے تواسنے دوپہر میں کتنے پھول اٹھاے؟	2	4	5	8	flowers	flowers	flowers	flowers	3 + <5> = 8	3					3	5	+	B	wordwindow-stone-blue.png	wordwindow-stone-green.png		
+en-US	9	2	2	6	"ली ने अपने घर के सामने 4 बिल्लियाँ देखी। वह सोने चली गई और जब वह सोकर उठी तो उसके घर के सामने 7 बिल्लियाँ थी। सोते समय उसके घर मे कितनी बिल्लियाँ आई?"	کاشفہ نے اپنے گھر کے سامنے 4 بلیاں دیکھی. وہ سونے چلی گئی تھی اور جب وہ اٹھی، اس کے گھر کے سامنے میں 7 بلیاں تھیں. جب وہ سو رہی تھی، تب کتنی بلیاں اس کے گھر میں آئ؟	2	3	4	5	cats	cats	cats	cats	4 + <3> = 7	2					4	3	+	B	wordwindow-stone-blue.png	wordwindow-stone-red.png				
+en-US	9	2	3	6	"मुन्नी की पेल मे कुछ शख थी। वह 2 और शख पाती है। अब उसके पास कुल 8 शख है। उसके पास पहले से कितने शख थे?"	منّی کی بالٹی میں کچھ سیپیاں تھیں- اسکو 2 اور سیپیاں ملیں- اب اس میں پاس 8 سیپیاں ہیں- شروع میں اسکے پاس کتنی سیپیاں تھیں؟	2	4	6	8
```
Comment (Shailin) [inline localized/games/wordwindow/wordwindow_level_ur.tsv]: Replace `सैली` with Urdu name

File: localized/games/eggquiz/eggquizmath_levels_ur.tsv [hunk 12]
```diff
@@ -178,46 +178,46 @@ PostTest_7	1	7001, 7002~7003, 7004, 7005, 7006~7007, 7008, 7009, 7010~7011, 7012
...
-			7011	word_problem	ہینری كے پاس 4 اسٹیمپس ہیں. ایرک كے پاس ہینری سے 5 اسٹیمپس زیادہ ہیں. ایرک كے پاس کتنے اسٹیمپس ہیں؟	हेनरी के पास 4 डाक टिकट हैं। एरिक के पास हेनरी की तुलना में 5 अधिक डाक टिकट हैं। एरिक के पास कितने डाक टिकट हैं?				8				
-			7012	word_problem	ہینری كے پاس 4 اسٹیمپس ہیں . ایرک كے پاس ہینری سے 5 زیادہ ڈال اسٹیمپس ہیں . ایرک كے پاس کتنے ستماپس ہیں ؟	एक पेड़ की शाखा पर 8 पक्षी बैठे हैं। कुछ उड़ गए। अब शाखा पर 3 पक्षी हैं। कितने पक्षी उड़ गए?				3				
+			7010	word_problem	صابرہ نے صبح 5 مچھلیاں پکڑی. اس نے دوپہر میں 4 اور مچھلیاں پکڑی. صابرہ نے سب ملکر کتنی مچھلیاں پکڑی؟	साबिरा ने सुबह 5 मछलियाँ पकड़ीं। उसने दोपहर में 4 और मछलियाँ पकड़ीं। साबिरा ने कुल कितनी मछली पकड़ी?				9				
+			7011	word_problem	ہاشم كے پاس 4 اسٹیمپس ہیں. اکرم كے پاس ہاشم سے 5 اسٹیمپس زیادہ ہیں. اکرم كے پاس کتنے اسٹیمپس ہیں؟	हाशिम के पास 4 डाक टिकट हैं। अकरम के पास हाशिम की तुलना में 5 अधिक डाक टिकट हैं। अकरम के पास कितने डाक टिकट हैं?				8				
+			7012	word_problem	ہاشم كے پاس 4 اسٹیمپس ہیں . اکرم كے پاس ہاشم سے 5 زیادہ ڈال اسٹیمپس ہیں . اکرم كے پاس کتنے ستماپس ہیں ؟	एक पेड़ की शाखा पर 8 पक्षी बैठे हैं। कुछ उड़ गए। अब शाखा पर 3 पक्षी हैं। कितने पक्षी उड़ गए?				3
```
Comment (bikashtudu) [inline localized/games/eggquiz/eggquizmath_levels_ur.tsv]: image

File: localized/games/wordwindow/wordwindow_level_ur.tsv [hunk 13]
```diff
@@ -1,241 +1,241 @@
...
+en-US	9	2	1	6	"तूबा ने सुबह 3 फूल उठाए। उसने दोपहर मे कुछ और फूल उठाए। यदि तूबा ने सभी मे 8 फूल उठाए, तो उसने दोपहर मे कितने उठाए?"	طوبا نے صبح میں 3 پھول اٹھاے. اسنے دوپہر میں کچھ پھول اور اٹھاے. اگر اسنے کل ملا کر  8 پھول اٹھاے تواسنے دوپہر میں کتنے پھول اٹھاے؟	2	4	5	8	flowers	flowers	flowers	flowers	3 + <5> = 8	3					3	5	+	B	wordwindow-stone-blue.png	wordwindow-stone-green.png		
+en-US	9	2	2	6	"ली ने अपने घर के सामने 4 बिल्लियाँ देखी। वह सोने चली गई और जब वह सोकर उठी तो उसके घर के सामने 7 बिल्लियाँ थी। सोते समय उसके घर मे कितनी बिल्लियाँ आई?"	کاشفہ نے اپنے گھر کے سامنے 4 بلیاں دیکھی. وہ سونے چلی گئی تھی اور جب وہ اٹھی، اس کے گھر کے سامنے میں 7 بلیاں تھیں. جب وہ سو رہی تھی، تب کتنی بلیاں اس کے گھر میں آئ؟	2	3	4	5	cats	cats	cats	cats	4 + <3> = 7	2					4	3	+	B	wordwindow-stone-blue.png	wordwindow-stone-red.png				
+en-US	9	2	3	6	"मुन्नी की पेल मे कुछ शख थी। वह 2 और शख पाती है। अब उसके पास कुल 8 शख है। उसके पास पहले से कितने शख थे?"	منّی کی بالٹی میں کچھ سیپیاں تھیں- اسکو 2 اور سیپیاں ملیں- اب اس میں پاس 8 سیپیاں ہیں- شروع میں اسکے
```
Comment (bikashtudu) [inline localized/games/wordwindow/wordwindow_level_ur.tsv]: Urdu spelling and transcript

File: localized/games/eggquiz/eggquizmath_levels_ur.tsv [hunk 14]
```diff
@@ -388,37 +388,37 @@ SpecialTest_21	1	210001, 210002, 210003, 210003, 210004
...
-			230001	word_problem	لیری کی 9 سال کی عمر ہے. اس کی بہن جولی ہے 6 سال کی عمر. کتنا بڑی اس کی بہن جولی کے مقابلے میں لیری ہے?	लुसी 9 साल की है। उसकी बहन जूली 6 साल की है। लुसी अपनी बहन जूली से कितनी बड़ी है?				3
-			230002	word_problem	سام نے 7 کوکیز اور جانے نے 2 کوکیز کھائی . سام کی تُلْنا میں جانے نے کتنی کوکیز کم کھائی ؟	सैम ने 7 बिस्कुट और जेन ने 2 बिस्कुट खाए। सैम की तुलना में जेन ने कितने कम बिस्कुट खाए?				5
-			230003	word_problem	امی سکول جانے كے لیے 4 بلوک چلتی ہے . جانے سکول جانے كے لیے 2 بلوک چلتی ہے . امی جانے کی تُلْنا میں کتنا زیادہ چلتی ہے ؟	एमी 4 ब्लोक चलती है स्कूल के लिए। जेन स्कूल के लिए 2  ब्लोक चलता है। एमी जेन की तुलना में स्कूल जाने के लिए और कितने  ब्लोक चलती हैं?				2
+			230001	word_problem	لیری کی 9 سال کی عمر ہے. اس کی بہن جوہی ہے 6 سال کی عمر. کتنا بڑی اس کی بہن جوہی کے مقابلے میں لیری ہے?	लुसी 9 साल की है। उसकी बहन जूही 6 साल की है। लुसी अपनी बहन जूही से कितनी बड़ी है?				3
+			230002	word_problem	سام نے 7 کوکیز اور جانے نے 2 کوکیز کھائی . سام کی تُلْنا میں جانے نے کتنی کوکیز کم کھائی ؟	सलमान ने 7 बिस्कुट और जाफ़री ने 2 बिस्कुट खाए। सलमान की तुलना में जाफ़री ने कितने कम बिस्कुट खाए?				5
```
Comment (bikashtudu) [inline localized/games/eggquiz/eggquizmath_levels_ur.tsv]: Urdu transcript

File: localized/games/eggquiz/eggquizmath_levels_ur.tsv [hunk 15]
```diff
@@ -388,37 +388,37 @@ SpecialTest_21	1	210001, 210002, 210003, 210003, 210004
...
 SpecialTest_23	1	230001~230010, 230001~230010, 230001~230010, 230001~230010, 230001~230010
-			230001	word_problem	لیری کی 9 سال کی عمر ہے. اس کی بہن جولی ہے 6 سال کی عمر. کتنا بڑی اس کی بہن جولی کے مقابلے میں لیری ہے?	लुसी 9 साल की है। उसकी बहन जूली 6 साल की है। लुसी अपनी बहन जूली से कितनी बड़ी है?				3
-			230002	word_problem	سام نے 7 کوکیز اور جانے نے 2 کوکیز کھائی . سام کی تُلْنا میں جانے نے کتنی کوکیز کم کھائی ؟	सैम ने 7 बिस्कुट और जेन ने 2 बिस्कुट खाए। सैम की तुलना में जेन ने कितने कम बिस्कुट खाए?				5
-			230003	word_problem	امی سکول جانے كے لیے 4 بلوک چلتی ہے . جانے سکول جانے كے لیے 2 بلوک چلتی ہے . امی جانے کی تُلْنا میں کتنا زیادہ چلتی ہے ؟	एमी 4 ब्लोक चलती है स्कूल के लिए। जेन स्कूल के लिए 2  ब्लोक चलता है। एमी जेन की तुलना में स्कूल जाने के लिए और कितने  ब्लोक चलती हैं?				2
+			230001	word_problem	آسمہ کی 9 سال کی عمر ہے. اس کی بہن جوہی ہے 6 سال کی عمر. کتنا بڑی اس کی بہن جوہی کے مقابلے میں آسمہ ہے?	लुसी 9 साल की है। उसकी बहन जूही 6 साल की है। लुसी अपनी बहन जूही से कितनी बड़ी है?				3
```
Comment (Shailin) [inline localized/games/eggquiz/eggquizmath_levels_ur.tsv]: Change the word `लुसी`. Check all the files and resolve similar issues

File: localized/games/wordwindow/wordwindow_level_ur.tsv [hunk 16]
```diff
@@ -1,241 +1,241 @@
...
+en-US	9	2	1	6	"तूबा ने सुबह 3 फूल उठाए। उसने दोपहर मे कुछ और फूल उठाए। यदि तूबा ने सभी मे 8 फूल उठाए, तो उसने दोपहर मे कितने उठाए?"	طوبا نے صبح میں 3 پھول اٹھاے. اسنے دوپہر میں کچھ پھول اور اٹھاے. اگر اسنے کل ملا کر  8 پھول اٹھاے تواسنے دوپہر میں کتنے پھول اٹھاے؟	2	4	5	8	flowers	flowers	flowers	flowers	3 + <5> = 8	3					3	5	+	B	wordwindow-stone-blue.png	wordwindow-stone-green.png		
+en-US	9	2	2	6	"ली ने अपने घर के सामने 4 बिल्लियाँ देखी। वह सोने चली गई और जब वह सोकर उठी तो उसके घर के सामने 7 बिल्लियाँ थी। सोते समय उसके घर मे कितनी बिल्लियाँ आई?"	کاشفہ نے اپنے گھر کے سامنے 4 بلیاں دیکھی. وہ سونے چلی گئی تھی اور جب وہ اٹھی، اس کے گھر کے سامنے میں 7 بلیاں تھیں. جب وہ سو رہی تھی، تب کتنی بلیاں اس کے گھر میں آئ؟	2	3	4	5	cats	cats	cats	cats	4 + <3> = 7	2					4	3	+	B	wordwindow-stone-blue.png	wordwindow-stone-red.png				
+en-US	9	2	3	6	"मुन्नी की पेल मे कुछ शख थी। वह 2 और शख पाती है। अब उसके पास कुल 8 शख है। उसके पास पहले से कितने शख थे?"	منّی کی بالٹی میں کچھ سیپیاں تھیں- اسکو 2 اور سیپیاں ملیں- اب اس میں پاس 8 سیپیاں ہیں- شرو
```
Comment (Shailin) [inline localized/games/wordwindow/wordwindow_level_ur.tsv]: `सारा` word is not updated

Comment (Shailin) [inline localized/games/wordwindow/wordwindow_level_ur.tsv]: Urdu text missing. Add Urdu text from the Excel file

Comment (Shailin) [inline localized/games/wordwindow/wordwindow_level_ur.tsv]: `सैडी` word is not updated

File: localized/games/eggquiz/eggquizmath_levels_ur.tsv [hunk 17]
```diff
@@ -161,13 +161,13 @@ MiniTest_7_2	1	72001, 72002, 72003, 72004, 72005
 			72002	operations_drag	11~19+11~19=?	regroupingfalse	horizontal			20~30				
 			72003	operations_drag	20-1~10=?		horizontal			1~10				
 			72004	compare_number_magnitudes	descending	اعداد کو بڑے سے چھوٹے کی ترتیب دیں	सबसे बड़े से लेकर सबसे छोटे तक के क्रम में संख्या को रखे			51~99				
-			72005	word_problem	بلی كے پاس 2 مربلیس ہیں . اینڈرے كے پاس بلی سے 3 زیادہ مربلیس ہیں . اینڈرے كے پاس کل کتنے مربلیس ہیں ؟	बिली के पास two मार्बल्स हैं। ऑड्रे के पास बिली की तुलना में three अधिक मार्बल्स हैं। ऑड्रे के पास कितने मार्बल्स हैं?				5				
+			72005	word_problem	بختیار كے پاس 2 مربلیس ہیں . علیزہ كے پاس بختیار سے 3 زیادہ مربلیس ہیں . علیزہ كے پاس کل کتنے مربلیس ہیں ؟	बख़्तियार के पास two मार्बल्स हैं। अलीज़ा के पास बख़्तियार की तुलना में three अधिक मार्बल्स हैं। अलीज़ा के पास कितने मार्बल्स हैं?				5				
 MiniTest_7_3	1	73001, 73002, 73003, 73004, 73005
```
Comment (ankitk20) [inline localized/games/eggquiz/eggquizmath_levels_ur.tsv:165]: replace two, three, four, etc with 2, 3, 4, etc respectively in TTS speech statements.. Check line 164 and 170

File: localized/games/wordwindow/wordwindow_level_ur.tsv (hunk 16 above)
Comment (ankitk20) [reply localized/games/wordwindow/wordwindow_level_ur.tsv]: Please fix this issue @bikashtudu

Comment (ankitk20) [inline localized/games/wordwindow/wordwindow_level_ur.tsv]: Remove shree

Comment (ankitk20) [inline localized/games/wordwindow/wordwindow_level_ur.tsv:236]: It is muaz i think. Just verify it and update if required.

**Review by ankitk20 — CHANGES_REQUESTED** [review]

**Review by Shailin — CHANGES_REQUESTED** [review]

**Review by Shailin — DISMISSED** [review]

**Review by ankitk20 — APPROVED** [review]

**Pull request merged**